javascript/src/piranha.js

```diff
@@ -0,0 +1,138 @@
+// Contributed by Ananye Agarwal (anag004) and Sarthak Behera (s7rthak). The authors are from the Indian Institute of Technology, Delhi (IIT Delhi). 
```

Comment:
Absolutely happy to list this!

That said, as an Uber OSS requirement, every JS file in this PR should have the Uber copyright block and link to the license at the top (above this line). See the Java example [here](https://github.com/uber/piranha/blob/master/java/piranha/src/main/java/com/uber/piranha/PiranhaUtils.java#L1)

Reply:
Done a8ce1614a7d21ea3ca8b542891847a6746ef550d